Research/ResearchWarcrimes.md

```diff
@@ -0,0 +1,47 @@
+
+2. To make said methods have each of their own difficulty on making them.
+
+3. To make said methods be able to be used in combinations together.
+
+4. To encourage co-operation between other departments and the marines.
+
+## Non-goals
+
+1. Significantly affect balance over a certain time period. This design does not aim to change the balance of the round in a quick manner, said methods will always take time to create, giving the antagonists time to not worry about it.
+
+2. Make the USCM dependant on it. Research is intended to be a side project just like OT and is going to treated as a side mini goal for the USCM.
+
+3. Make said features easily available. Due to their power factor, these features/methods should have a certain rarity to them to avoid them affecting the rounds too much.
+
+## Content
+
+The features added would be, a new way that makes chemicals work by applying it differently via TOUCH (through spraying and gas) and making said chemicals now also work on xenomorphs (depending if the property can do it). 
+New additional properties that also contain the new features of TOUCH application for more variety.
+New code of chemicals to use the new mechanics of properties (example, Epinephrine's unique intravenous feature has now been replaced by a new property that does the same thing but now it can be added to other chemicals with the help of the new property) 
+Tweak the cost of properties based on the new features to make them less/more available (Stimulant property costs increased while underused properties such as organ healing ones decreased) 
+Tweak chemical generation to make CL 1/2 chemicals have more unique properties so researchers always find something new than the same property in 5 different generated chemicals.
+Change of code by replacing it with flags and defines to make things less..spaghetti code I guess.
+Buying clearence levels now slightly increasing passive income so you get rewarded by buying clearence yourself than getting it for free by someone else like the CL/Survivors.
+Disabling redundant/impossible to find chemicals generating in recipes so you don't need to basically pray to the RNG gods for that one chemical.
+Decrease the cost of buying chemical papers so they can be worthwile to get.
+Numerous fixes that have been bugging the department for some time.
+
+## Potential Changes
+
+1. Tweak the properties to be less/more powerful (In case of certain properties being way too OP or barely useful)
+
+2. Tweak spray bottle code to provide more chemicals per "injection" (In case if injecting chemicals by using spray bottles is seen too weak, however should be still weaker than using pills and syringes)
+
+3. Provide some way for xenos to slow down Research indirectly (In case if time won't be enough for xenomorphs to deal with it)
+
+4. Include more ways to obtain CL 3+ chemicals (In case there are severe lack of CL 3+ chemical papers/vials available on maps)
```

Comment:
Potential changes is not a category to list changes that you plan to make.
Remove this section if you don't know how to use it properly, it's not necessary anyways